addons/l10n_co/__manifest__.py

```diff
@@ -17,7 +17,7 @@
     'website': 'https://www.odoo.com/colombia',
     'depends': [
         'account',
-        'base_address_city',
+        'base_address_extended',
```

Comment:
This dependency is weird, because the cities are loaded in l10n_co_edi (because of the extra field on them)

Reply:
moved to l10n_co_edi